lib/smart_answer_flows/register-a-birth/outcomes/oru_result.govspeak.erb

```diff
@@ -203,8 +203,8 @@
 
   Service | Fee
   -|-
-  Register a birth | £105
-  Copy of a birth registration certificate | £65
+  Register a birth | <%= format_money(reg_data_query.register_a_birth_fees.register_a_birth) %>
+  Copy of a birth registration certificate | <%= format_money(reg_data_query.register_a_birth_fees.copy_of_birth_registration_certificate) %>
```

Comment:
@ikennaokpala I've extracted a fees partial as you suggested. See 8203e2bf5125816043ae46b6a5fc953ca4b4e3ec (in PR #2424).